Port create subarray dataset

astrobc1 opened the pull request:
@zonca I think this one is good to go!

zonca commented on liger_iris_pipeline/tests/test_create_subarray_dataset.ipynb:
do we want to remove this file?

astrobc1 replied:
We can delete:
`run_image2_subarray.ipynb` (second function in `test_image2.py`)
`test_dark_subarray.ipynb`
`test_merge_subarrays.ipynb`
`test_create_subarray_dataset.ipynb`
`test_preprocess_flatfield.ipynb`

I now see `test_flat.ipynb` applies the Step to the sci data and is different from `test_preprocess_flatfield.py`. Should I refactor that one quickly?

zonca replied:
Yes, they are two different algorithms, one is to prepare a flat field and the other one is to apply it p

zonca reviewed (approved):
just remove the notebook

astrobc1 commented:
I refactored test_flat and deleted all of the jupyter notebooks in [port_flatfield](https://github.com/oirlab/liger_iris_pipeline/tree/port_flatfield). Let's review+merge that after create subarray dataset is merged?
